static/sass/_pattern_application.scss

```diff
@@ -15,7 +15,8 @@
           font-size: 1rem;
           font-weight: 500;
           height: 2rem !important;
-          line-height: 1.9rem !important;
+          line-height: 1.6rem !important;
+          border:  3px solid transparent;
```

Comment:
adding a transparent border by default will avoid having the wrong line-hight when border is not shown.